eng/scripts/dump_process.ps1

```diff
@@ -3,7 +3,7 @@ Set-Location $args[0]
 $timestamp = $(get-date -f MM-dd-HH-mm)
 
 while ($true) {
```

Comment:
@dougbu Does this run every during the build to help diagnose hangs?

Reply:
Yup. Have a look at the log artifacts for the Windows jobs. Since the file is repeatedly overwritten, it should end with the list of processes just before AzDO gives up on us. Just a little something @JunTaoLuo cooked up 😃

Reply:
Great! We need it cross plat as well 😄